src/next-child-view-container.js

```diff
@@ -0,0 +1,108 @@
+import _ from 'underscore';
+import emulateCollection from './utils/emulate-collection';
+
+// Provide a container to store, retrieve and
+// shut down child views.
+const Container = function() {
+  this._views = [];
```

Comment:
cool. doing the https://github.com/joefitter/backbone.enumerable/blob/master/src/backbone.enumerable.js#L9 approach.